website/docs/r/policy_definition.html.markdown

```diff
@@ -67,6 +69,8 @@ The following arguments are supported:
 
 * `description` - (Optional) The description of the policy definition.
 
+* `management_group_id` - (Optional) the management group id (IE: `${data.azurerm_management_group.test.group_id`) where where to define this policy
```

Comment:
We tend to phrase this as:

```suggestion
* `management_group_id` - (Optional) The ID of the Management Group where this policy should be defined. Changing this forces a new resource to be created.
```

Reply:
I'll update as suggested, but want to leave in the example. I was confused myself at first because `azurerm_management_group` has both an `.id` and a `.group_id`. 

Alternatively, I could modify it to something like "The Group ID (and *not* the ID) of the Management Group.."

Thoughts?

Reply:
If you'd like to add that information i would add it in a notes box under that line, or to the example at the top of this page

Reply:
ok. cool.